.travis.yml

```diff
@@ -7,6 +7,9 @@ node_js:
   - 0.12
 before_install:
   - "test $TRAVIS_NODE_VERSION = '0.6' && npm install -g npm@1.3.26 || npm install -g npm"
+script:
+  - ./node_modules/.bin/eslint mustache.js
```

Comment:
Sorry about not commenting that... It's temporary too debug the v0.6 error on Travis. The first builds didnt print any errors, it just exited with status 1. After some debugging v0.6 / npm v1.3.26 locally I found that calling `eslint` directly, instead of via `npm test`, actually printed the error. It was some wierd error from a third party module..

As I'm not able too reproduce it locally anymore, I'll revert this line and keep my fingers crossed as Travis runs again ;)